src/Uno.UI/UI/Xaml/FrameworkElement.iOS.cs

```diff
@@ -65,11 +65,7 @@ public override void LayoutSubviews()
 
 						Rect finalRect;
 						var parent = Superview;
-						if (parent is UIElement
-							|| parent is ISetLayoutSlots
-							// In the case of ListViewItem inside native list, its parent's parent is ListViewBaseInternalContainer
-							|| parent?.Superview is ISetLayoutSlots
```

Comment:
This change looks like it's going to undo https://github.com/unoplatform/uno/commit/b3c0a7bdabb4d11d967906c9bad28f3e68ce8719. It's likely going to break https://github.com/unoplatform/uno/blob/fec43e75cd0624a397190b3b37b2763542174115/src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_UIElement.TransformToVisual.cs#L163.

Reply:
said test is still passing.

Reply:
Okay, This did broke several tests using TransformToVisual.
These tests was initially passing, but later started to fail after a rebase on feb 2 which was puzzling..:
- initial: https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=67831&view=ms.vss-test-web.build-test-results-tab&runId=2220128&resultId=100000&paneView=debug
- later: https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=68134&view=ms.vss-test-web.build-test-results-tab&runId=2224502&resultId=100000&paneView=debug

That said, I've updated TransformToVisual to compensate for the changes.